docs/mp/guides/40_performance-tuning.adoc

```diff
@@ -0,0 +1,95 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2022 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Performance Tuning
+:h1Prefix: MP
+:description: Helidon MP Performance Tuning
+:feature-name: Performance Tuning
+:microprofile-bundle: true
+= :keywords: helidon, microprofile, micro-profile, performance, tuning
+
+
+== Basic tuning for Helidon MP
+
+1. Use `io.helidon.microprofile.bundles:helidon-microprofile-core` dependency instead of full (`helidon-microprofile`) dependency.
+2. Configure the size of the Netty worker thread pool
+3. Configure the size of Helidon's server thread pool
+4. Configure Netty's maxOrder setting if using Helidon 2.4.1 or earlier
+5. Configure/tune Java heap size as per any Java application
```

Comment:
We should either expand on this, or remove it for now.

Reply:
Applied.